tests/test_manifest.py

```diff
@@ -761,3 +761,32 @@ def test_create_manifests(
                 data_model_labels="class_label",
             )
             assert all_results == expected_result
+
+    def test_get_record_based_manifest_with_files(self, helpers):
```

Comment:
@linglp I don't think a non-integration test really does much here, so I think it's fine as is.

Reply:
Although @GiaJordan the test you crated doesn't call `updateDatasetManifestFiles`, is that intentional?

Reply:
@andrewelamb yes, I went back and forth for a while on how to structure this test. In the end since it was more integration than unit I tried to mirror what has happening in `manifest/commands.py` with the creation of a `ManifestGenerator` object and calling `get_manifest`

Reply:
@andrewelamb @linglp also note I decided to add a test case for file based manifests as well so that's been changed

Reply:
@GiaJordan Is the `updateDatasetManifestFiles` method directly tested currently?

Reply:
@andrewelamb it doesn't look like it